helpers/tistream.py

```diff
@@ -0,0 +1,21 @@
+from StringIO import StringIO
```

Comment:
Please write a header comment what the class does. Also don't forget copyright lines as in other files